code/frameworks/html-vite/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@storybook/html-vite",
+  "version": "7.0.0-alpha.56",
+  "description": "Storybook for HTML and Vite: Develop HTML in isolation with Hot Reloading.",
+  "keywords": [
+    "storybook"
+  ],
+  "homepage": "https://github.com/storybookjs/storybook/tree/main/frameworks/html-vite",
+  "bugs": {
+    "url": "https://github.com/storybookjs/storybook/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybookjs/storybook.git",
+    "directory": "frameworks/html-vite"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/storybook"
+  },
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./preset": {
+      "require": "./dist/preset.js",
+      "import": "./dist/preset.mjs",
+      "types": "./dist/preset.d.ts"
+    },
+    "./package.json": {
+      "require": "./package.json",
+      "import": "./package.json",
+      "types": "./package.json"
+    }
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "types/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "check": "tsc --noEmit",
+    "prep": "../../../scripts/prepare/bundle.ts"
+  },
+  "dependencies": {
+    "@storybook/addons": "7.0.0-alpha.56",
+    "@storybook/builder-vite": "7.0.0-alpha.56",
+    "@storybook/channel-postmessage": "7.0.0-alpha.56",
+    "@storybook/channel-websocket": "7.0.0-alpha.56",
+    "@storybook/client-api": "7.0.0-alpha.56",
+    "@storybook/core-server": "7.0.0-alpha.56",
+    "@storybook/html": "7.0.0-alpha.56",
+    "@storybook/node-logger": "7.0.0-alpha.56",
+    "@storybook/preview-web": "7.0.0-alpha.56",
+    "magic-string": "^0.26.1",
+    "vite": "3"
```

Comment:
We'll need to open this up to allow version 4, but we can do that in another PR, for all the vite frameworks.